client/src/pages/OrderScreen.js

```diff
@@ -331,9 +339,9 @@ function OrderScreen() {
                 <DollarAmount
                   size={"medium"}
                   label={"Order total"}
-                  amount={
+                  amount={(
                     subtotal +
-                    subtotal * 0.01 * tipPercent.replace(/\D/g, "")
+                    subtotal * 0.01 * tipPercent.replace(/\D/g, "")).toFixed(2)
```

Comment:
any reason we're not using the formatter? tbh totally fine as long as it works

Reply:
oh i'm not sure, this is what the totals on the rest of the page are doing so i just referenced that (these were just a couple values that got missed)